src/index.css

```diff
@@ -1,27 +1,82 @@
 body {
+  font: 100% helvetica, sans-serif;
+  color: #333;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-size: 100%;
+}
+
+.calcCenter {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 0 auto;
+  padding: 20px;
+  padding-top: 50px;
+  border-radius: 0;
+  width: 100%;
+  background: linear-gradient(90deg, #00c9ff 0%, #92fe9d 100%);
+  height: 100vh;
+}
+
+.calcCenter .calcCenter h1 {
+  font-size: 3rem;
```

Comment:
- [ ] Good job so far with your calculator design :+1:  But It looks a bit different from the Figma template :cry: Please ensure that your calculator component matches with the provided design because as a developer you need to always follow the design provide you your future employer will judge you on that. :smile: 

#### Given design
<img width="558" alt="calculator (3)" src="https://user-images.githubusercontent.com/6615394/190138271-9269ed20-679f-4414-8fda-37381457a51a.png">

#### Your design
![Screenshot from 2022-09-14 10-46-41](https://user-images.githubusercontent.com/6615394/190138404-63314d8e-ca9c-4ad5-a4db-52ce28a80d6e.png)